fe/fe-core/src/main/java/org/apache/doris/catalog/SparkResource.java

```diff
@@ -243,6 +264,16 @@ protected void setProperties(Map<String, String> properties) throws DdlException
         return sparkConfigs;
```

Comment:
```suggestion
        return sparkConfig;
```
The method name also need changed?